docs/guide/piece-pages.md

```diff
@@ -1,257 +1,162 @@
----
-title: "Piece Pages"
----
+# Piece index and show pages
 
-# Piece Pages
+**Piece index pages** allow editors to add pages listing pieces of a particular type. They also allow pieces to have their own web pages -- known as **show pages** in Apostrophe. Show pages are automatically available once an index page and pieces of the matching piece type are created.
```

Comment:
**"add pages listing pieces"** is really tripping me up. We're talking about a display page for a all of the items and a template for a single item display page, right?

"They also allow pieces to have their own web pages -- known as **show pages**" -- I think we should remove this.

Reply:
"Index pages list pieces of a particular type."

Reply:
"Once an index page is added, each piece will get its own show page..."